fetch/api/basic/error-after-response.html

```diff
@@ -0,0 +1,34 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Fetch: network timeout after receiving the HTTP response headers</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="../resources/utils.js"></script>
+  </head>
+  <body>
+    <div id="log"></div>
+    <script>
+function checkReader(test, reader, checkRead)
```

Comment:
I think this would be a lot clearer if it took a lambda instead of a boolean, so it was `checkReader(test, response.body.getReader(), r => r.read())` and `r => r.closed` at the call sites.